modules/api_management/variables.tf

```diff
@@ -80,6 +80,20 @@ variable "azurerm_api_management_diagnostic_settings" {
   }
 }
 
+variable "alert_rules_settings" {
+  type = list(object({
+    name          = string,
+    severity      = string,       # possible values: Sev0, Sev1, Sev2, Sev3 or Sev4
+    frequency     = string,       # frequency in ISO-8601 eg. PT1M -> 1 per Month
+    detector_type = string,       # possible values: FailureAnomaliesDetector, RequestPerformanceDegradationDetector, DependencyPerformanceDegradationDetector, ExceptionVolumeChangedDetector, TraceSeverityDetector, MemoryLeakDetector
```

Comment:
This comment is very long, maybe provide a link to the possible values?

Reply:
Ok, provided the link to terraform.registery